.github/workflows/preview.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to Preview
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    needs: [build]
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.6.6
```

Comment:
I think if you use `2.6` it should use the latest patch version

Reply:
👍